diff --git a/integrations/expressive-code/theming.ts b/integrations/expressive-code/theming.ts
index e207aac3895cf5dd41dcef13bed2b1345b1e18ac..19121a26ea7c2bc6088e6748e30e946587761679 100644
--- a/integrations/expressive-code/theming.ts
+++ b/integrations/expressive-code/theming.ts
@@ -22,7 +22,7 @@ export function preprocessThemes(
 		// If the current entry is the name of a bundled theme, load it
 		if (theme === 'starlight-dark' || theme === 'starlight-light') {
 			const bundledTheme = theme === 'starlight-dark' ? nightOwlDark : nightOwlLight;
-			return customizeBundledTheme(ExpressiveCodeTheme.fromJSONString(bundledTheme));
+			return customizeBundledTheme(ExpressiveCodeTheme.fromJSONString(JSON.stringify(bundledTheme)));
 		}
 		// Otherwise, just pass it through
 		return theme;
